### PR TITLE
Embedding bert class

### PR DESCRIPTION
**Pull request by elliegarcia**
I wrote the bert embedding wrapper. It doesn't seem to work with the DBSCAN clustering method because of a difference in the expected dimensions of the embeddings. I am not sure if that is a problem because from the bert embedding class or from the parameters set for DBSCAN.

### CODE REVIEW

File: experiments/embedding_methods/bert_embedding.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+class BertTransformerEmbedding(EmbeddingInterface):
+
+    def __init__(self, model_name: str):
+        self.model = BertModel.from_pretrained(model_name)
+        self.model_name = model_name
+        self.tokenizer = BertTokenizer.from_pretrained(model_name)
+
+    def embed_sentence(self, sentence: str):
+        """
```
Comment (samuelgoodall) [inline experiments/embedding_methods/bert_embedding.py]: this does not embed a sentence with bert. it tokenizes the input sentence(aka.) splits it into smaller chunks for example "koalabear"-> ["ko","ala", "bear"] then you generated an array that got you the ids belonging to those wordpieces. you didnt call the actual model

**Review by samuelgoodall — CHANGES_REQUESTED** [review]

File: experiments/embedding_methods/bert_embedding.py (hunk 1 above)
Comment (samuelgoodall) [reply experiments/embedding_methods/bert_embedding.py]: https://mccormickml.com/2019/05/14/BERT-word-embeddings-tutorial/

Comment (samuelgoodall) [reply experiments/embedding_methods/bert_embedding.py]: https://towardsdatascience.com/how-to-use-bert-from-the-hugging-face-transformer-library-d373a22b0209

Comment (elliegarcia) [reply experiments/embedding_methods/bert_embedding.py]: I updated the script to reflect the changes we talked about, but every time I try to run it my computer crashes. I think it's taking up too much memory on my computer, or I did something wrong?

**Pull request merged**